modules/gapi/test/streaming/gapi_streaming_vpl_core_test.cpp

```diff
@@ -327,6 +329,72 @@ TEST(OneVPL_Source_CPU_FrameAdapter, InitFrameAdapter)
     EXPECT_TRUE(0 == surf->get_locks_count());
 }
 
+TEST(OneVPL_Source_Default_Source_With_OCL_Backend, Accuracy)
+{
+    using namespace cv::gapi::wip::onevpl;
+
+    auto create_from_string = [](const std::string& line){
+        std::string::size_type name_endline_pos = line.find(':');
+        std::string name = line.substr(0, name_endline_pos);
+        std::string value = line.substr(name_endline_pos + 1);
+        return CfgParam::create(name, value);
+    };
+
+    std::vector<CfgParam> source_cfgs;
+    source_cfgs.push_back(create_from_string("mfxImplDescription.AccelerationMode:MFX_ACCEL_MODE_VIA_D3D11"));
```

Comment:
i suppose it should be simple way, like:  CfgParam::create()

please feel free to ignore it: comment not to apply